### PR TITLE
Fix/birth e2e tests

### PR DESCRIPTION
**Pull request by makelicious**
1. Update E2E tests to match right id. 
2. Update copies to match changes (remove duplicates)
3. Remove extra parenthesis from conditional

### CODE REVIEW

File: src/form/addresses/index.ts [hunk 1]
```diff
@@ -144,7 +144,7 @@ export const defaultAddressConfiguration: IAddressConfiguration[] = [
             `(${detailsDontExist} || ${mothersDetailsDontExistOnOtherPage}) || ${hideIfMotherAddressNotAvailable[0].expression}`
           ),
           expressionToConditional(
-            `(${detailsDontExist} || ${mothersDetailsDontExistOnOtherPage}) || ${hideIfMotherAddressNotAvailable[0].expression} || !${primaryAddressSameAsOtherPrimaryAddress})`,
```
Comment (makelicious) [inline src/form/addresses/index.ts:147]: It seems that during the merge from `countryconfig` to `farajaland` an extra condition was added. Should this be added to 1.5 release or is this specific for `farajaland`?

Comment (Zangetsu101) [reply src/form/addresses/index.ts:147]: @makelicious Could you please check if the same issue is in farajaland release-v1.5 branch? If so let's amend it there as well

Comment (Zangetsu101) [reply src/form/addresses/index.ts:147]: The change was farajaland specific I think. I had encountered merge conflicts when pulling in changes from countryconfig. The malformed bracket was most likely from that :sweat_smile:

Comment (Zangetsu101) [reply src/form/addresses/index.ts:147]: it seems to be fine on the release branch at least https://github.com/opencrvs/opencrvs-farajaland/pull/1000/files#diff-84362893885c6ba15e20d5582fa9468ed65c2b1c9e7a2bdbd9f3c14cd3f2f860R146-R148

**Review by Zangetsu101 — APPROVED** [review]

File: src/form/addresses/index.ts (hunk 1 above)
Comment (makelicious) [reply src/form/addresses/index.ts:147]: Thank you for helping out!

**Pull request merged**